src/Common/Visuals/Avalonia/MorganStanley.ComposeUI.Common.VisualUtils/DispatcherAdapter.cs

```diff
@@ -0,0 +1,49 @@
+﻿/// ********************************************************************************************************
+///
+/// Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License").
+/// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+/// See the NOTICE file distributed with this work for additional information regarding copyright ownership.
+/// Unless required by applicable law or agreed to in writing, software distributed under the License
+/// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and limitations under the License.
+/// 
+/// ********************************************************************************************************
+
+using Avalonia.Threading;
+using NP.Utilities.Attributes;
```

Comment:
Please remove unused imports and package references until they are actually needed.

Reply:
removed the file altogether